apps/playground/.eslintrc.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
   root: true,
-  extends: ["custom"],
+  extends: ['custom'],
+  rules: {
+    'jsx-a11y/alt-text': 'off',
+    '@next/next/no-img-element': 'off',
```

Comment:
Was using `yarn lint` quite a bit and these weren't helpful in our case. Although what's up with this `custom` config? Couldn't find any info about it except the source code on npm. Why don't we copy paste it instead of getting on npm?

Reply:
We are not getting it from npm. `custom` refers to the local package in the monorepo (have a look at packages/eslint-config-custom), so the overrides should be made there.

This is the setup that came with turborepo.

Reply:
Updated the actual config package instead